psi4/src/psi4/scfgrad/jk_grad.cc

```diff
@@ -1524,27 +1482,37 @@ void DFJKGrad::compute_hessian()
                 int ny = 3 * Ncenter + 1;
                 int nz = 3 * Ncenter + 2;
 
-                size_t stride = static_cast<size_t> (Pncart) * Mncart * Nncart;
+                size_t stride = static_cast<size_t>(Pncart) * Mncart * Nncart;
```

Comment:
`stride` still needed?

Reply:
Good catch!  I think there will be quite a few of these kinds of loop variables that are not needed, now that the new buffer structure is in place.  They will be optimized away so they're harmless.  If I see a lot of compiler warnings I'll go take a pass through and get rid of them.

Reply:
Lori removed all those I found already, most superfluous `stride`/`ncart` should be gone. I did a quick 'find all' search while reviewing.